src/components/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,48 @@
+import { ComponentPropsWithoutRef, useState } from "react";
+
+export interface TabConfig {
+  label: string;
+  content: string;
+}
+
+export interface TabsProps extends ComponentPropsWithoutRef<"div"> {
+  tabsConfig: TabConfig[];
+}
+
+export function Tabs({ ...props }) {
+  const { tabsConfig, defaultIndex, ...rest } = props;
+  const [selectedIndex, setSelectedIndex] = useState(defaultIndex ?? 0);
+
+  function handleClick(index: number) {
```

Comment:
Personally, I use `const handleClick = () => {`.

But here, he use a `function handleClick() {`.

What do you prefer? @mazmassa

Reply:
The second one...